src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,94 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.AutoLogOutput;
+import org.littletonrobotics.junction.Logger;
+
+public class Gripper extends SubsystemBase {
+    private static Gripper INSTANCE;
+    private final GripperIO io;
+    public GripperIO.ControlMode controlMode;
+    private final GripperInputsAutoLogged inputs = GripperIO.inputs;
+
+    @AutoLogOutput private final Mechanism2d mechanism2d = new Mechanism2d(0, 0);
+
+    private final MechanismRoot2d root = mechanism2d.getRoot("Gripper", 0, 0);
+    private final MechanismLigament2d gripper =
```

Comment:
1. "gripper" is a bit general name.
2. It's not being updated anywhere.

Reply:
What other name do you suggest?

Reply:
f6fafa5931444c17e23a9565262df1f539a5477f

Reply:
This is not a relevant commit. A more specific name can be simply `gripperLigament`. The reasoning is that "gripper" can mean many things in a subsystem with the same name, so you'd want to be more clear in your naming.